Application/src/view/visuals/Draw.java

```diff
@@ -1,11 +1,16 @@
-package visuals;
+package view.visuals;
 
 import javafx.scene.canvas.Canvas;
 import javafx.scene.canvas.GraphicsContext;
 
 import javafx.scene.image.Image;
 
-import visuals.handlers.Animation;
+import view.DiagramView;
+import view.handlers.Animation;
+import view.visuals.component.ActorClass;
+import view.visuals.component.Coordinates;
+import view.visuals.component.DiagramClass;
+import view.visuals.component.Message;
```

Comment:
Just import everything from view maybe?

Reply:
When i was moving stuff around it intelliJ did not recognize them if i tried to import everything through *. I can change it here on git though hoping it works?